src/v2/models/AppState.js

```diff
@@ -12,6 +12,7 @@
 
 import { _, Model } from 'okta';
 import Logger from 'util/Logger';
+import RemediationEnum from '../ion/RemediationEnum';
```

Comment:
we don't need to put Enum in the file name.. lets say  this `RemediationFormTypes`

Reply:
We could call it `RemediationFormNames` as they are not `type`. But I thought we might throw more constants into this file.

Reply:
`RemediationFormMetaData` `RemediationFormConstants` ?

Reply:
rename to `RemediationConstants`